gnomad_qc/v3/sample_qc/subpop_analysis.py

```diff
@@ -0,0 +1,217 @@
+import argparse
+import hail as hl
+import logging
+
+from gnomad.sample_qc.ancestry import (
+    assign_population_pcs,
```

Comment:
I'm assuming this will be the script for all subpop analysis, so we might need an `assign_population_pc`, For now let's either remove that, or add in the use of population assignment